Error when building with ninja

**Describe the bug (__must be provided__)**
Hello, This project is awesome and I'm really interested in it! But when I try to build it, I ran into trouble.

I found that I can't build it with ninja.(Maybe we have not support it now?)

**Your Environments (__must be provided__)**

* OS: `Linux  5.11.0-37-generic #41-Ubuntu  x86_64 GNU/Linux`
* Compliler: `g++ (Ubuntu 10.3.0-1ubuntu1) 10.3.0`
* CPU: `Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz`
* Commit id `cd2ecb922b`

**How To Reproduce(__must be provided__)**
1. `mkdir build && cd build`
2. `cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS=YES ../`

 **Expected behavior**

build success

**Additional context**
```
CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/home/jun/dev/cpp-projects/nebula/build' '-t' 'cleandead'

  failed with:

   ninja: error: build.ninja:207: bad $-escape (literal $ must be written as $$)
    COMMAND = cd /home/jun/dev/cpp-projects/nebula/build && $(MAKE) instal...
                                                            ^ near here



CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/home/jun/dev/cpp-projects/nebula/build' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:207: bad $-escape (literal $ must be written as $$)
    COMMAND = cd /home/jun/dev/cpp-projects/nebula/build && $(MAKE) instal...
                                                            ^ near here



CMake Generate step failed.  Build files cannot be regenerated correctly.
```
Also, I have found [robotology/robotology-superbuild/issues/83](https://github.com/robotology/robotology-superbuild/issues/83) with same issue, and they fix this in [68a218225804](https://github.com/robotology/ycm/commit/f2c61b1b9f5b2a7935bf79d1d43a68a218225804).

I guess it is a good way to start some contribution, and I'm willing to submit a patch for you(of course if it is a real bug...)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,7 +48,7 @@ include(NebulaCustomTargets)
 
 add_custom_target(
     install-all
-    COMMAND $(MAKE) install
+    COMMAND $MAKE install
     WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
 )
 

